docs/howtos/integrations/_langgraph_agent_evaluation.md

```diff
@@ -0,0 +1,550 @@
+# Building and Evaluating a ReAct Agent for Fetching Metal Prices
+
+AI agents are becoming increasingly valuable in domains like finance, e-commerce, and customer support. These agents can autonomously interact with APIs, retrieve real-time data, and perform tasks that align with user goals. Evaluating these agents is crucial to ensure they are effective, accurate, and responsive to different inputs.
+
+In this tutorial, we'll:
+
+1. Build a [ReAct agent](https://arxiv.org/abs/2210.03629) to fetch metal prices.
+2. Set up an evaluation pipeline to track key performance metrics.
+3. Run and assess the agent's effectiveness with different queries.
+
+Click the [link](https://colab.research.google.com/drive/1IsmUfCVD_pVQ7HGsKj97BXOZBGXTIF8m?usp=sharing) to open the notebook in Google Colab.
+
+## Prerequisites
+- Python 3.8+
+- OpenAI API key
```

Comment:
Note: depending on OpenAI is a bad idea. It's okay for now but we have to make sure that our integration pages are more accessible to people using different providers

this is something we are slowly working towards

Reply:
Got it